apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -153,7 +153,8 @@ class ProgressLesson extends React.Component {
 
     // There's no url for a lesson so use the url of the first level of the lesson
     // as the url for the lesson.
-    // TODO: Make the back-end support lesson urls and redirect to the first level.
+    // TODO: Make the back-end return a lesson url as part of the lesson metadata so we
+    // don't need to pass it separately from lesson here and in ProgressLessonTeacherInfo.
```

Comment:
Can you create a jira ticket and link it to a github comment for this TODO.

Reply:
Oh sorry, forgot to include the link here: https://codedotorg.atlassian.net/browse/LP-1634